packages/react-components/stories/dial/dialBase.stories.tsx

```diff
@@ -18,7 +21,9 @@ export default {
   },
 } as ComponentMeta<typeof DialBase>;
 
-export const Main: ComponentStory<typeof DialBase> = ({ yMin, yMax, showName, showUnit, ...args }) => (
+type StoryInputs = DialSettings & { alarmPoint?: DataPoint; propertyPoint?: DataPoint };
```

Comment:
Non-blocking feedback: The strict TS eslint settings you plan to introduce will prefer you use `interface` when typing records.

Reply:
This might be useful to you https://github.com/sindresorhus/type-fest/blob/main/source/spread.d.ts.